src/client/content_cao.cpp

```diff
@@ -171,6 +171,20 @@ static void updatePositionRecursive(scene::ISceneNode *node)
 	node->updateAbsolutePosition();
 }
 
+static bool logOnce(const std::ostringstream &from, std::ostream &log_to)
```

Comment:
This might also fit into `log.cpp` so that the Lua API could make use of it in a future change.

Reply:
Yeah everything that has similar code to this should be refactored in the future.